src/components/atoms/TextInput.vue

```diff
@@ -34,7 +34,7 @@
       <img
         class="focus:border-none"
         :src="icons[iconState]"
-        alt="Send Image."
+        alt="Send icon"
```

Comment:
This alt text can be translated

Reply:
it doesn't seem to be used but I can translate it anyway.

Reply:
Done